docs/kap_proposals/kap_6_hashicorp_vault.md

```diff
@@ -0,0 +1,84 @@
+# Hashicorp Vault  
+  
+This feature allows the user to fetch secrets from [Hashicorp Vault](https://www.vaultproject.io/), an online Kapitan Secret backend `vault`.  
+  
+Author: [@vaibahvk](https://github.com/vaibhavk) [@daminisatya](https://github.com/daminisatya)  
+## Specification  
+  
+The following variables need to be exported to the environment where you run this script in order to authenticate to your HashiCorp Vault instance:  
```

Comment:
Also, the way we configure secret backend variables is within the inventory, not env vars. e.g. the recipients for gpg or keys for gkms/awskms as part of `kapitan.secrets.<backend>`. Maybe it would make sense for the vault token but not much else.

Reply:
I have added the support to provide set these variables from within the inventory.